### PR TITLE
Update ingest

### PR DESCRIPTION
**Pull request by marcellp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 ===========
 
+An application for ingesting data into Biomage
+
+Set-up
+------
+
+Make sure you have Docker, docker-compose installed.
+Make sure your input folder contains the required files: `barcodes.tsv`, `genes.tsv`, `matrix.mtx`. Make sure your output folder is empty. These conditions will be checked by the script before it runs.
+
```
Comment (ivababukova) [inline README.md:11]: Also put "Make sure you have an input and output folders at the root of this project."

Comment (ivababukova) [reply README.md:11]: After that, you can delete steps 1. and 2. from the running instructions

File: input/meta.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "geoffrey-mok_east-anglia_old-chicken-data",
+    "organism": "ggallus",
+    "input": {
+        "type": "10x"
+    }
```
Comment (ivababukova) [inline input/meta.json:6]: should this be commited? it is dataset-specific

File: README.md (hunk 1 above)
Comment (marcellp) [reply README.md]: Fixed.

File: input/meta.json (hunk 2 above)
Comment (marcellp) [reply input/meta.json]: Fixed. Read new README.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,47 @@
...
+4. Go to S3 to `biomage-originals-production` bucket. You should see several folders.
+Each of them contains unprocessed experiment results for a user. Navigate to the correct
+folder with analysis files you want to preprocess and load in the platform. The folder
+should contain files `barcodes.tsv`, `genes.tsv`, `matrix.mtx`.
+
+5. Download the `barcodes.tsv`, `genes.tsv`, `matrix.mtx` files, located in S3 from the
+previous step. Make sure that they are saved inside the `input` folder you created in step 2.
+
+6. Fill out the `meta.json` file. This is the configuration that is being used to save
+and process the files appropriately.
```
Comment (ivababukova) [inline README.md:34]: add instructions on how to fill that file up

Comment (marcellp) [reply README.md:34]: instructions are directly below

**Review by ivababukova — APPROVED** [review]

**Pull request merged**